lib/osv/ecosystems.py

```diff
@@ -310,6 +310,36 @@ def enumerate_versions(self, package, introduced, fixed, limits=None):
     return self._get_affected_versions(versions, introduced, fixed, limits)
 
 
+class Debian(Ecosystem):
+  """Debian ecosystem"""
+
+  _API_PACKAGE_URL = 'https://snapshot.debian.org/mr/package/{package}/'
+  versions_to_idx = None
+
+  def sort_key(self, version):
+    if self.versions_to_idx is None:
+      raise Exception('versions to idx not initiated')
+
+    return self.versions_to_idx[version]
+
+  def enumerate_versions(self, package, introduced, fixed, limits=None):
```

Comment:
We will need to add the "introduced" version detection in this module here instead of in the converter.

We should keep Debian advisories as close to the source as possible, and put the heavy lifting into our "osv" module/infra.